bin/cli.js

```diff
@@ -324,10 +322,7 @@ For more information, see https://webpack.js.org/api/cli/.`);
 				}
 			});
 
-			if (
-				typeof outputOptions.colors === "undefined" &&
-				process.stdout.isTTY === true
-			)
+			if (typeof outputOptions.colors === "undefined")
```

Comment:
This was fixed most recently to not print on non.-tty interfaces

Reply:
Exactly! I am afraid there's going to be a regression...

Reply:
@ematipico There won’t be. As explained above, supports-color checks `process.stdout.isTTY` but it wasn’t working before because the library wasn’t used correctly.

Reply:
Thanks for the double reassurance! :)